### PR TITLE
Fix memory test

### PR DESCRIPTION
**Pull request by eivindjahren**
## Pre review checklist

- [x] Read through the code changes carefully after finishing work
- [x] Make sure tests pass locally (after every commit!)
- [ ] Prepare changes in small commits for more convenient review (optional)
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [ ] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: tests/performance_tests/test_memory_usage.py [hunk 1]
```diff
@@ -62,45 +61,38 @@ def mock_target_accessor() -> EnsembleAccessor:
...
+        experiment_id = storage.create_experiment(
+            parameters=ert.ensembleConfig().parameter_configuration
+        )
+        source = storage.create_ensemble(experiment_id, name="prior", ensemble_size=100)
+        gen_obs_keys = ert.getObservations().getTypedKeylist(
+            EnkfObservationImplementationType.GEN_OBS
+        )
+        observations = ert.getObservations()
+
+        ens_config = ert.ensembleConfig()
```
Comment (yngve-sk) [inline tests/performance_tests/test_memory_usage.py:75]: Might as well get+store this variable just below L65 to remove the dbl invocation

File: tests/performance_tests/test_memory_usage.py [hunk 2]
```diff
@@ -62,45 +61,38 @@ def mock_target_accessor() -> EnsembleAccessor:
...
+    path = Path(path) / "ensembles"
+    with open_storage(path, mode="w") as storage:
+        experiment_id = storage.create_experiment(
+            parameters=ert.ensembleConfig().parameter_configuration
+        )
+        source = storage.create_ensemble(experiment_id, name="prior", ensemble_size=100)
+        gen_obs_keys = ert.getObservations().getTypedKeylist(
+            EnkfObservationImplementationType.GEN_OBS
+        )
+        observations = ert.getObservations()
```
Comment (yngve-sk) [inline tests/performance_tests/test_memory_usage.py:73]: Can declare this before L70 to avoid dbl invocation

**Review by yngve-sk — APPROVED** [review]
LGTM, only minor comments 🚀

File: tests/performance_tests/test_memory_usage.py (hunk 2 above)
Comment (eivindjahren) [reply tests/performance_tests/test_memory_usage.py]: Done

File: tests/performance_tests/test_memory_usage.py (hunk 1 above)
Comment (eivindjahren) [reply tests/performance_tests/test_memory_usage.py]: Done

**Pull request merged**